Assets/Plugins/Editor/Uplift/LogHandler.cs

```diff
@@ -15,9 +15,17 @@ public class LogHandler : IDisposable
 		private string stack = "";
 
 		private StreamWriter OutputStream;
+		private LogType previouslyFilteredType;
 
-		public LogHandler(string fileName = "uplift.log", bool appendToCurrentLogFile = false, bool showStack = false)
+		public LogHandler(
+			string fileName = "uplift.log",
+			bool appendToCurrentLogFile = false,
+			bool showStack = false,
+			LogType logType = LogType.Log)
 		{
+			previouslyFilteredType = Debug.logger.filterLogType;
+			Debug.logger.filterLogType = logType;
```

Comment:
OK so you are changing all loggers during the time uplift is doing its work.
This also assumes nothing is happening in the background (e.g. threads).

Fine for now, but probably we should just rewrite this log handler by creating a special Logger instance with our file writer in it. It might be more logical.